app/javascript/components/Users/Edit/index.tsx

```diff
@@ -0,0 +1,81 @@
+import React from 'react';
+import { H2 } from '../../common/Headlines';
+import Input from '../../common/Input';
+import Button from '../../common/Button';
+
+type Props = {
+  authenticityToken: string;
+  errors: [string];
+};
+
+function FormErrors({ errors }) {
```

Comment:
Here, you need to specify the prop types, otherwise ESLint/TypeScript will probably complain. For this small component, you can also do it inline:

```suggestion
function FormErrors({ errors }: { errors: string[] }) {
```